src/swsssdk/dbconnector.py

```diff
@@ -104,6 +104,14 @@ def load_sonic_db_config(sonic_db_file_path=SONIC_DB_CONFIG_FILE):
             raise RuntimeError(msg)
         SonicDBConfig._sonic_db_config_init = True
 
+    @staticmethod
+    def isInit():
+        return SonicDBConfig._sonic_db_config_init
+
+    @staticmethod
+    def initialize(sonic_db_file_path=SONIC_DB_CONFIG_FILE):
```

Comment:
This function is not needed.
Check https://github.com/Azure/sonic-swss-common/blob/master/common/dbconnector.h#L42-L49. `load_sonic_db_config` will be mapped to `initialize`, so you can always `load_sonic_db_config` in python code no matter which library you are using.

Reply:
Removed initialize function